my-project/src/LanguageBox.jsx

```diff
@@ -0,0 +1,108 @@
+import { useState, useEffect, useContext } from 'react';
+import { Octokit } from '@octokit/rest';
+import './App.css'; // Optional: For styling
+import { MyContext } from './MyContext';
+
+const LanguageBox = () => {
+  // const { myState, setMyState } = useContext(MyContext); // This state is created for setting the access token
+  const [languages, setLanguages] = useState([]);
+  const [selectedLanguage, setSelectedLanguage] = useState('');
+  const [repos, setRepos] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [message, setMessage] = useState('');
+  //setting the accesstoken in environment variable
+  const apiToken = import.meta.env.VITE_API_KEY;
+  console.log(apiToken);
```

Comment:
always remember to remove console when you submit your PR.

Reply:
okay